src/convertToObject.js

```diff
@@ -6,7 +6,48 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const arrSoursString = sourceString;
+  const result = {};
+
+  for (let i = 0; i < arrSoursString.length; i++) {
+    if (arrSoursString[i] === ':') {
+      result[`${suchKey(arrSoursString, i)}`] = such(arrSoursString, i);
+    }
+  }
+
+  return result;
+}
+
+function such(arr, index) {
+  let word = '';
+
+  for (let i = index; i < arr.length; i++) {
+    if (arr[i] === '\n') {
+      word = word + arr[i];
+    }
+
+    if ('abcdefghijklmnopqrstuvwxyz- ,1234567890!.#%()/'.includes(arr[i])) {
+      word = word + arr[i];
```

Comment:
The character check in the `if` condition is not comprehensive. Consider using a regular expression to match valid CSS property values.